### PR TITLE
Update to eslint 2.0.0

### PR DESCRIPTION
**Pull request by willyaranda**
Hi there!

I am working with latest eslint and wanted to use this as a plugin. This is a PR to fix it for the upcoming 2.0 of ESlint.

It should have a major bump of version number. But that can be done once #48 is done (which I can happily try to do).

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -3,8 +3,8 @@
         "node": true
     },
 
-    "ecmaFeatures": {
-        "globalReturn": false
+    "parserOptions": {
+        "ecmaVersion": 6,
```
Comment (lo1tuma) [inline .eslintrc]: Currently this project still uses ES5. Also I really want to keep `globalReturn` disabled.

File: .eslintrc [hunk 2]
```diff
@@ -23,9 +23,6 @@
         "no-dupe-keys": 2,
         "no-dupe-args": 2,
         "no-else-return": 2,
-        "no-empty": 2,
-        "no-empty-character-class": 2,
```
Comment (lo1tuma) [inline .eslintrc]: `no-empty` and `no-empty-character-class` still exist is eslint 2. What is the reason to remove the config of those rules here?

File: .eslintrc [hunk 3]
```diff
@@ -23,9 +23,6 @@
...
-        "no-empty-label": 2,
```
Comment (lo1tuma) [inline .eslintrc]: According to the changelog this rule has been replaced by `no-labels`  with {"allowLoop": true, "allowSwitch": true} option.

**Comment by lo1tuma** [comment]
Thanks for the Pull Request.

Most of the changes looking good so far, I’ve only spotted a few minor problems.
I don’t plan to release a new major version based on a release candidate. I could only offer to release a beta or release candidate version until ESLint 2.0 is out.

**Comment by willyaranda** [comment]
ESlint 2 has just been released. I updated the PR with the comments you made. I hope you find this useful.

**Comment by lo1tuma** [comment]
Awesome. Thank you!

:beers:

**Pull request merged**